microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -86,16 +85,84 @@ The capability to back up and restore the {product-title} database is now availa
 //[id="microshift-4-14-storage"]
 //=== Storage
 
+[id="microshift-4-14-running-apps"]
+=== Running Applications
+//still need to add
+
+[id="microshift-backup-and-restore"]
+=== Backup and restore
+The capability to back up and restore the {product-title} database is now available. You can manually back up and restore data on all supported systems at any time. See the xref:../microshift_backup_and_restore/microshift-backup-and-restore.adoc#microshift-backup-and-restore[Backing up and restoring {product-title} data] documentation for more information.
+
 //[id="microshift-4-14-lvms-system-requirements"]
 //==== LVMS system requirements
 
-//[id="microshift-4-14-deprecated-removed-features"]
-//== Deprecated and removed features
+[id="microshift-4-14-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+// Note: use [discrete] for these sub-headings.
+
+[discrete]
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
 
-//[id="microshift-4-14-bug-fixes"]
-//== Bug fixes
+Networking updates to {product-title} {product-version} include traffic flow pattern, gateway, and custom configuration changes.
 
-//etc
+[discrete]
+[id="microshift-4-14-traffic-flow-change"]
+==== North-south traffic flow changed
+
+The external gateway bridge and the physical device on the host are no longer connected. The north-south traffic between the network service and the OVN external switch flows from the host kernel to {product-title} through the external gateway bridge. See the xref:../microshift_networking/microshift-networking.adoc#microshift-description-connections-network-topology_microshift-networking[MicroShift networking] documentation for more information.
+
+[id="microshift-4-14-deprecated-and-removed"]
+== Deprecated and removed features
+
+Some features available in previous releases of {product-title} have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the tables below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _Available_
+
+* _Deprecated_
+
+* _Removed_
+
+.{product-title} deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 | 4.14
+
+|Network configuration flags
+|-
+|-
+|Deprecated
+
+|CIDR notation
+|-
+|-
+|Removed
+
+|====
+[discrete]
+[id="microshift-4-14-network-config-flags-deprecated"]
+==== Network configuration flags are deprecated
```

Comment:
These flags are actually removed from 4.14. That's my mistake. They aren't still supported.